com.unity.render-pipelines.universal/Shaders/Utils/ScreenSpaceShadows.shader

```diff
@@ -53,15 +49,14 @@ Shader "Hidden/Universal Render Pipeline/ScreenSpaceShadows"
         {
             UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
-            float deviceDepth = SAMPLE_TEXTURE2D_X(_CameraDepthTexture, sampler_CameraDepthTexture, input.uv.xy).r;
-
 #if UNITY_REVERSED_Z
-            deviceDepth = 1 - deviceDepth;
+            float deviceDepth = SAMPLE_TEXTURE2D_X(_CameraDepthTexture, sampler_CameraDepthTexture, input.uv.xy).r;
+#else
+            float deviceDepth = SAMPLE_TEXTURE2D_X(_CameraDepthTexture, sampler_CameraDepthTexture, input.uv.xy).r;
+            deviceDepth = deviceDepth * 2.0 - 1.0;
```

Comment:
Are you using UNITY_REVERSED_Z as a shortcut to detect non-GL platforms?

Reply:
Would `#if defined(SHADER_API_GLCORE) || defined(SHADER_API_GLES) || defined(SHADER_API_GLES3)` be equivalent but more correct?

Reply:
Yes, it is general to use it to detect platform instead of using platform specification defines.

Reply:
I used 'SHADER_API_GLCORE' before, but @phi-lira  suggested me to use 'UNITY_REVERSED_Z'.

Reply:
I see!